src/tui/tui.go

```diff
@@ -5,6 +5,8 @@ import (
 	"os"
 	"strconv"
 	"time"
+
+	"github.com/gdamore/tcell"
```

Comment:
This makes non-Windows binaries, which don't need tcell, unnecessarily larger.

```
-rwxr-xr-x  1 jg  staff  2785696 Sep 29 17:29 fzf-darwin_amd64
-rwxr-xr-x  1 jg  staff  3198592 Sep 29 17:28 fzf-darwin_amd64-tcell-linked
```

An easy, albeit non-elegant, solution would be to define it as a non-member variable in `tcell.go` (similarly to `_screen` https://github.com/junegunn/fzf/blob/f4f47f5fe32db500765b47d937f5eb8fa27be130/src/tui/tcell.go#L123-L125),  because FullscreenRenderer is effectively a singleton.

Reply:
Thanks for pointing it out, I didn't realize it.